mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -50,69 +53,88 @@ class ElasticsearchStore(AbstractStore):
 
     def __init__(self, store_uri: str = None, artifact_uri: str = None) -> None:
         self.is_plugin = True
-        connections.create_connection(hosts=[urllib.parse.urlparse(store_uri).netloc])
-        ElasticExperiment.init()
-        ElasticRun.init()
+        self.es = Elasticsearch([urllib.parse.urlparse(store_uri).netloc],
+                                verify_certs=True)
+        self.indices = IndicesClient(self.es)
+        ExperimentIndex.init(indices=self.indices)
+        RunIndex.init(indices=self.indices)
         super(ElasticsearchStore, self).__init__()
 
     def _hit_to_mlflow_experiment(self, hit: Any) -> Experiment:
-        return Experiment(experiment_id=hit.meta.id, name=hit.name,
-                          artifact_location=hit.artifact_location,
-                          lifecycle_stage=hit.lifecycle_stage)
+        return Experiment(experiment_id=hit["_id"], name=hit["_source"]["name"],
+                          artifact_location=hit["_source"]["artifact_location"],
```

Comment:
wondering if it is normal for client code to access an underscored field, seems weird according to python coding rules

Reply:
The object elasticsearch returns is made that way. If I want to access my object I have to look into _source.